modules/database/src/ioc/dbtemplate/dbtoolsIocRegister.c

```diff
@@ -15,23 +15,24 @@
 /* dbLoadTemplate */
 static const iocshArg dbLoadTemplateArg0 = {"filename", iocshArgStringPath};
 static const iocshArg dbLoadTemplateArg1 = {"var1=value1,var2=value2", iocshArgString};
-static const iocshArg * const dbLoadTemplateArgs[2] = {
-    &dbLoadTemplateArg0, &dbLoadTemplateArg1
-};
+static const iocshArg dbLoadTemplateArg2 = {"path1:path2:...", iocshArgString};
+static const iocshArg *const dbLoadTemplateArgs[3] = {
+    &dbLoadTemplateArg0, &dbLoadTemplateArg1, &dbLoadTemplateArg2};
 static const iocshFuncDef dbLoadTemplateFuncDef = {
     "dbLoadTemplate",
-    2,
+    3,
     dbLoadTemplateArgs,
     "Load the substitution file given as first argument, apply the substitutions\n"
     "for each template in the substitution file, and load them using 'dbLoadRecords'.\n\n"
     "The second argument provides extra variables to substitute in the\n"
-    "template files (not the substitution file).\n\n"
+    "template files (not the substitution file). The third argument provides\n"
+    "a list of paths to search through for the subsitution and template files.\n\n"
     "See 'help dbLoadRecords' for more information.\n\n"
-    "Example: dbLoadTemplate db/my.substitutions 'user=myself,host=myhost'\n",
+    "Example: dbLoadTemplate db/my.substitutions 'user=myself,host=myhost' 'path/to/subst:path2/to2/subst2'\n",
```

Comment:
How to document the addition of this argument?

If this was a C prototype, this would be a doxygen tag `\since UNRELEASED`.  I am unsure what to do about an IOC shell function through.  Maybe just add a line `Third argument added in UNRELEASED.`?

Reply:
In the documentation - yes. (+1)

But here, in the online help?
If you're on an older system, you don't get the hint about the new functionality.
If you're on a new system, the introduction date is not that valuable when working with the system.

Reply:
Should I adjust the help message then or leave it as is?